bin/templates/project/app/build.gradle

```diff
@@ -22,33 +22,32 @@ apply plugin: 'com.android.application'
 buildscript {
     repositories {
         mavenCentral()
-        maven {
-            url "https://maven.google.com"
-        }
+        google()
         jcenter()
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.1.0'
+        classpath 'com.android.tools.build:gradle:3.2.0'
     }
 }
 
 // Allow plugins to declare Maven dependencies via build-extras.gradle.
 allprojects {
     repositories {
-        mavenCentral();
+        mavenCentral()
```

Comment:
Add google() or replace mavenCentral() with google() here, as suggested in <https://github.com/apache/cordova-android/pull/441/files/478450fe4fe6a7b06e15d2b68facf0aecd63a27e#r228693626>?

Reply:
What has #441 to do with this?

Reply:
Updates and suggestions in #441 may be relevant here. This PR is to update Gradle in general. #441 was raised to introduce Kotlin support but it did include some general updates that I think we should consider outside of adding the Kotlin support.